app/src/components/market/steps/items/summary_step.tsx

```diff
@@ -81,7 +82,12 @@ class SummaryStep extends Component<Props> {
         )}
         {marketMakerAddress && <div>Market Maker deployed at {marketMakerAddress}</div>}
         <Button onClick={this.back}>Back</Button>
-        <Button onClick={this.submit}>Create Market</Button>
+        <Button
+          disabled={status !== Status.Ready && status !== Status.Done && status !== Status.Error}
```

Comment:
I don't think it should be enabled again after it's done.

Reply:
I remove the validation for the status `Done`